src/main/fc/rc.c

```diff
@@ -462,7 +462,7 @@ FAST_CODE_NOINLINE void rcSmoothingSetFilterCutoffs(rcSmoothingFilter_t *smoothi
     // update or initialize the derivative filter
     oldCutoff = smoothingData->derivativeCutoffFrequency;
     if ((rcSmoothingData.derivativeFilterType != RC_SMOOTHING_DERIVATIVE_OFF)
-        && (currentPidProfile->ff_interpolate_sp == FF_INTERPOLATE_OFF)
+        && (currentPidProfile->ff_interpolate_sp != FF_INTERPOLATE_OFF)
```

Comment:
The logic was inverted, preventing any updating of the derivative filter from the initial values.  Should have been if not interpolation off.  Fixed.